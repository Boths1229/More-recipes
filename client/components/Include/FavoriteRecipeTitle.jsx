import React from 'react';

const FavoriteRecipeTitle = () => ((
  <div className="row">
    <section className="col-md-12">
      <h1 className="title">
        Favorite recipes
      </h1>
      <div>
        <hr />
      </div>
    </section>
  </div>
));

export default FavoriteRecipeTitle;
