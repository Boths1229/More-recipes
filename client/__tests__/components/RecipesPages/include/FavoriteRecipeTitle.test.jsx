import React from 'react';
import { shallow } from 'enzyme';
import FavoriteRecipeTitle from '../../../../components/Include/FavoriteRecipeTitle';

describe('Favorite recipe title', () => {
  it('matches snapshot', () => {
    const output = shallow(<FavoriteRecipeTitle />);
    expect(output).toMatchSnapshot();
  });
});
