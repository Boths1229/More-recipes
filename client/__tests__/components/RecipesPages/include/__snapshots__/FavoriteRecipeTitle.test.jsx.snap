// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Favorite recipe title matches snapshot 1`] = `
<div
  className="row"
>
  <section
    className="col-md-12"
  >
    <h1
      className="title"
    >
      Favorite recipes
    </h1>
    <div>
      <hr />
    </div>
  </section>
</div>
`;
